Model/Animal.cs

```diff
@@ -0,0 +1,73 @@
+﻿    public class Animal
+    {
+    internal const ushort MAX_AGE = 2000;
+    internal const float MIN_SIZE = 0.03f;
+    internal const float MAX_SIZE = 33.0f;
+    internal const float MAX_JUMP_LENGTH = 5.25f;
+    internal  const byte MAX_FROG_AGE = 17;
```

Comment:
```suggestion
    internal const byte MAX_FROG_AGE = 17;
```
и такое dotnet format пофиксит. а нейминг - если ему подкинуть красивый [.editorconfig](https://docs.microsoft.com/ru-ru/visualstudio/ide/create-portable-custom-editor-options?view=vs-2022)